lib/inline-imports.js

```diff
@@ -89,15 +89,15 @@ function inlineImports(fileContents, options = {}, visited = new Set()) {
 module.exports = inlineImports;
 module.exports.lineToImports = function (fileContents, options = {}, visited = new Set()) {
   const result = new Map();
-  for (let { line, match, lineNumber } of linesWithInlinedImportsOf(
+  for (let { line, match, lineNumber, resolvedPath } of linesWithInlinedImportsOf(
     fileContents,
     options,
     visited,
   )) {
     // We're only interested in the inlined import lines, ignore any
     // non-matching lines
     if (match) {
-      result.set(lineNumber, line);
+      result.set(lineNumber, { resolvedPath, line });
```

Comment:
Could we use `filename` ?

Reply:
Sure, I can do that. It will actually be the full file path (not just base name) but that's the same as in `fragmentDefinition.loc.filename` so I can carry the `filename` terminology throughout.

Reply:
Done